index_release.go

```diff
@@ -0,0 +1,23 @@
+package versionbundle
+
+import (
+	"time"
+)
+
+const indexReleaseTimestampFormat = "2006-01-02T15:04:05.00Z"
+
+type IndexRelease struct {
```

Comment:
I am not happy with the naming. I just chose something to get going. Please suggest alternatives.

Reply:
- `ReleaseBundle`
- `ReleaseEntry`
- `Release` - I actually like this the most. Makes members sound right: `release.Authorities` or `if release.Active {`.

Reply:
We already have a `Release` type which is what the `cluster-service` "aggregates".

Reply:
But when we don't create releases by aggregation anymore, could we refactor that (i.e. remove old and related stuff and use new one)?

Reply:
Possible that we can throw away the other types when we got rid of the aggregation magic in the end. That would be cool.